src/Headers.ts

```diff
@@ -64,7 +71,7 @@ export default class HeadersPolyfill {
    * Returns a `ByteString` sequence of all the values of a header with a given name.
    */
   get(name: string): string | null {
-    return this[NORMALIZED_HEADERS][normalizeHeaderName(name)] || null
+    return this[NORMALIZED_HEADERS][normalizeHeaderName(name)] ?? null
```

Comment:
When headers are set to an `''` value, we should still return that value, and not null

Reply:
Oh, I see. It does capture a broader set of falsey values but given the header value strict type, we should be safe. Thanks for explaining this!